### PR TITLE
Add pagination support on CHost's errata tab

### PR DESCRIPTION
**Pull request by swadeley**
Hello

I need this for a pagination test. I will post test results with the new test when its ready.

Thank you

### CODE REVIEW

**Comment by swadeley** [comment]
Hello

This is the test:

[ Test chost errata pagination #8173 ](https://github.com/SatelliteQE/robottelo/pull/8173)

**Review by mirekdlugosz — COMMENTED** [review]
black doesn't like your line breaks and fails Travis. Please fix.
Also, it seems to me we could drop two of these new widget definitions, as they are already included in Pagination and you don't use them in Robottelo PR.

**Review by lhellebr — APPROVED** [review]
LGTM and since the test passes: pending only Mirek's comments and Travis.

File: airgun/views/contenthost.py [hunk 1]
```diff
@@ -244,6 +244,14 @@ class errata(SatTab):
                 'Id': Text('./a'),
             },
         )
+        select_all = Checkbox(
+            locator=".//input[@type='checkbox'][@ng-change='allSelected()']")
+        items_per_page = Select(
+            locator=".//select[@ng-model='table.params.per_page']")
+        total_packages = Text(
+            "//span[@class='pagination-pf-items-total ng-binding']")
```
[the remark this answers is not in the document]
Comment (swadeley) [reply airgun/views/contenthost.py]: OK, removing now

File: airgun/views/contenthost.py [hunk 2]
```diff
@@ -244,6 +244,14 @@ class errata(SatTab):
                 'Id': Text('./a'),
             },
         )
+        select_all = Checkbox(
+            locator=".//input[@type='checkbox'][@ng-change='allSelected()']")
+        items_per_page = Select(
+            locator=".//select[@ng-model='table.params.per_page']")
```
[the remark this answers is not in the document]
Comment (swadeley) [reply airgun/views/contenthost.py]: OK, removing now

**Review by mirekdlugosz — APPROVED** [review]
ACK, but not merging until Travis results are in and all checks are passing.

**Comment by swadeley** [comment]
Thank you

**Pull request merged**